src/sentry/db/models/fields/slug.py

```diff
@@ -0,0 +1,13 @@
+from django.core.exceptions import ValidationError
+from django.db.models import SlugField
+
+
+def no_numeric_validator(value: str) -> None:
+    if value.isnumeric():
```

Comment:
I see @schew2381 used a regex pattern [here](https://github.com/getsentry/sentry/blob/master/src/sentry/api/fields/sentry_slug.py#L18) instead of general numeric which doesn't include all cases of numeric. I'm ok with this change but when he did the migration he made sure we don't have anything numeric using the pattern. Therefore we might have existing slugs that could fail this validation.

Reply:
I think @ykamo001 is making sure all the numeric slugs are gone before running the migration

Reply:
Also almost forgot this but we should use `isdecimal` instead, it validates a much smaller subset of things as seen [here](https://stackoverflow.com/a/42250941/19354746).
```suggestion
    if value.isdecimal():
```

Reply:
We're using the same regex on both sides now. Sorry, this was outdated from stale PR